src/clients/evm/types.ts

```diff
@@ -52,6 +53,13 @@ export type Strategy = {
     signerAddress: string,
     data: Propose | Vote
   ): Promise<string>;
+  getVotingPower(
+    strategyAddress: string,
+    voterAddress: string,
+    block: number,
```

Comment:
Replace with timestamp to match.
Read from strategy or fallback to currentBlock - 1.